exercises/bob/testlib/TesterMain.f90

```diff
@@ -0,0 +1,207 @@
+!------------------------------------------------------------------
+!------------------------------------------------------------------
+! MIT License
+!
+! Copyright (c) 2019 Exercism
+!
+! Permission is hereby granted, free of charge, to any person obtaining a copy
+! of this software and associated documentation files (the "Software"), to deal
+! in the Software without restriction, including without limitation the rights
+! to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+! copies of the Software, and to permit persons to whom the Software is
+! furnished to do so, subject to the following conditions:
+
+! The above copyright notice and this permission notice shall be included in all
+! copies or substantial portions of the Software.
+
+! THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+! IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+! FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+! AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+! LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+! OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+! SOFTWARE.
+!------------------------------------------------------------------
+!------------------------------------------------------------------
+!
```

Comment:
I believe the license portion is covered in the source root of this repository.  We don't need to include it in this file.

Reply:
The source file in the root directory is not distributed to the students, so it makes sense to have it in these library files. I don't have a strong opinion on this, though.

Reply:
I do get that... but I don't think we do this for other tracks, either.  The signing up for the website gives the licensing terms, terms of use, etc, and implies agreement, I think.

Since Exercism is the one distributing the software to the student (and the student is not transmitting the software (the tests) back, I don't think it applies to them to have it included in their code.

Reply:
I think if this is important, rather than having it in the test file, we can place the licensing information in the readme that is delivered to the student.  Better than having it duplicated in every test file.  The Readme can be built to include other information in addition to the information specific to the exercise.  If there is a change in the license terms, we would have to change it in every file.  Just doesn't feel right here.

Reply:
I don't think it's necessary to distribute the license file to students, whether in the tests or in the README, as they've signed up for the site and agreed to the terms.

I am not a lawyer, but if somehow we determine that this is something we need to do, then I want it to be done consistently across all the tracks, not only in this track. If we need to do it consistently, then we can do it without duplicating the actual text into the individual files.

Reply:
I am fine with deleting the license stuff in the tester lib. The lib is not really made for students to edit, it is only for executing the tests.